rest-api-spec/src/main/resources/rest-api-spec/api/security.get_api_key.json

```diff
@@ -40,6 +40,11 @@
         "type":"boolean",
         "default":false,
         "description":"flag to query API keys owned by the currently authenticated user"
+      },
+      "with_limited_by":{
+        "type":"boolean",
+        "default":false,
+        "description": "flag to show the limited-by role descriptors of API Keys"
       }
```

Comment:
@elastic/clients-team This PR adds a new query parameter to both GetApiKey and QueryApiKey APIs.